README.md

```diff
@@ -116,8 +116,9 @@ Once you have updated your dataSchema, commit your changes, merge to main/master
 - ```entityTableName``` - the name of the table in your database; a string; mandatory
 - ```description``` - a meta description of the table, which can be a blank string: ```''```; mandatory
 - ```keys``` - an array of objects which determines how dfe-analytics-dataform will transform each of the fields in the table. Each table listed within ```dataSchema``` has its own ```keys``` i.e. : ```dataSchema: [{entityTableName: '', description: '', keys: {}}, {entityTableName: '', description: '', keys: {}}, {entityTableName: '', description: '', keys: {}}...]```.
-- If the table has a primary key that is not ```id```, then this object may optionally have the attribute ```primary_key``` (containing the name of the field that is the primary key, if it is not ```'id'```.).
-- If you wish to create an assertion which fails if no Create, Update or Delete events have been received in the last ```dataFreshnessDays``` days for this entity, then this object may optionally include an integer-valued ```dataFreshnessDays``` parameter.
+- ```primary_key``` - optional; if the table has a primary key that is not ```id```, then this should contain the name of the field that is the primary key, if it is not ```'id'```.
+- ```dataFreshnessDays``` - optional; if set, creates an assertion which fails if no Create, Update or Delete events have been received in the last ```dataFreshnessDays``` days for this entity.
+- ```materialisation``` - optional; may be ```'view'``` or ```'table'```. Defaults to 'table' if not set. Determines whether the ```entity_version```, ```entity_latest``` and ```entity_field_updates``` tables for this entity will be materialised by Dataform as views or tables. Recommended usage is to set this to ```'table'``` if these tables will be used more than once a day, or ```'view'``` if not to save query costs.
```

Comment:
Would it be helpful to add `materialisation` to the `example.js` or to the `bat_apply_test.js`? - I can see that `dataFreshnessDays` has been added to `example.js`

Reply:
Good idea - I've done this

Reply:
Ignore this, I can see it has been added to example.js